PSKoans/Koans/Constructs and Patterns/AboutCustomObjects.Koans.ps1

```diff
@@ -33,19 +33,19 @@ Describe '[PSCustomObject]' {
     It 'can be built by trimming objects down' {
         $Object = Get-ChildItem -Path $Home | Select-Object -First 1 -Property Name, Parent
         $Object | Should -BeOfType PSCustomObject
-        __ | Should -Be $Object.Parent.Name
+        '__' | Should -Be $Object.Parent.Name
     }
 
     It 'can have arbitrary properties' {
-        $Object = [PSCustomObject]@{ '__' = 'Enter Property Name' }
+        $Object = [PSCustomObject]@{ '__' = 'PropertyValue' }
 
-        __ | Should -Be $Object.PSObject.Properties.Count
-        __.PSObject.Properties.Name | Should -Be 'PropertyName'
+        '__' | Should -Be $Object.PSObject.Properties.Count
```

Comment:
We don't want quotes here; the expected value is numeric, and while Pester _might_ do the conversion for us I wouldn't rely on it.

(Also, if it's not clear; `__` is an alias exported from PSKoans that points to `Get-Blank`, which will generally cause almost any assertion performed to fail if its output is passed to `Should`.)

Reply:
Fixed.

Thanks for the note on the alias, I had missed that.